lib/tasks/locale.rake

```diff
@@ -207,6 +221,9 @@ namespace :locale do
     system('rmsgmerge', '--sort-by-msgid', '--no-fuzzy-matching', '-o', Rails.root.join('locale', 'en', 'manageiq-all.po').to_s, Rails.root.join('locale', 'en', 'manageiq.po').to_s, Rails.root.join('locale', 'manageiq.pot').to_s)
     system('mv', '-v', Rails.root.join('locale', 'en', 'manageiq-all.po').to_s, Rails.root.join('locale', 'en', 'manageiq.po').to_s)
     system('rm', '-rf', tmp_dir)
+
+    remove_line_numbers(Rails.root.join('locale/manageiq.pot'))
+    remove_line_numbers(Rails.root.join('locale/en/manageiq.po'))
```

Comment:
I do want to get this in as I think removing line numbers will help see the actual changes to all of these files.

Reply:
@Frank-NB Can you verify if it's ok for us to remove line numbers in the translations too?

Reply:
I can definitely do that but it should happen automatically.  That is, I'd expect the translators to take the .pot files and generate the .po from it, so those line numbers are driven by the .pot.  (Though we can do a first pass for diff purposes manually of course, but it doesn't have to be this PR)

Reply:
Exactly, I'd like @Frank-NB's opinion as I'm unsure how they create/update the translation files.

Reply:
Hi @jrafanie , Yes, we take the .pot files and generate tho .po form it, so I think if you update en/manageiq.po and manageiq.pot, translations will come back without line numbers. but I think we need to test it after this PR merged, hope it can work as design.

Reply:
Considering they are only comments, this should not affect anything from working.  We had considered removing the comments entirely, but I think having at least the files is useful diagnostically.